recipes/newrelic/infrastructure/ohi/mongodb/debian.yml

```diff
@@ -252,14 +282,14 @@ install:
               read -r -p "MongoDB Port? (Default 27017) " MONGO_PORT
               MONGO_PORT=${MONGO_PORT:-27017}
```

Comment:
Thank you. Started to work on it on Friday. As this week's hero duties permit, I'll get back to finish correcting it.

Reply:
Hi @gsanchezgavier - Addressed the non-default issue you reported in 36f8ac48b5cb3a5c06aa5fad67c6a8470bf59b9e and 14eb91356d0e9129062bd4784c94e36ce8681bde and tested it for RHEL/Debian for both, SCRAM and TSL auth scenarios. Any port passed to the `install` command to with the `NR_CLI_DB_PORT` envar will be picked up, otherwise that envar defaults to `27017`

Reply:
Thanks! changes looks good @amolero-nr